src/exchanges/gemini-client.js

```diff
@@ -59,12 +68,14 @@ class GeminiClient extends EventEmitter {
     if (!subscription) {
       subscription = {
         market,
-        wss: this._connect(remote_id),
+        wss: this._connect(remote_id, mode),
         lastMessage: undefined,
         reconnectIntervalHandle: undefined,
         remoteId: remote_id,
         trades: false,
         level2Updates: false,
+        tickers: false,
+        mode
```

Comment:
I had to add the `mode` property here so that when `_reconnect` is called it can know whether to add the `&top_of_book=true` when reconnecting a tickers sub